### DISCUSSION TITLE
Ant design v5' requirements

### DISCUSSION THREAD

**Discussion by daydarkln** [ant-design/ant-design#48672] [Q&A]
I've got an app with such [package.json](https://gist.github.com/daydarkln/ab2046c11ff2eff63f933d5b62d38b71#file-package-json)

And when I try to up this app, it stucks on errors:

```
ERROR in ./node_modules/scroll-into-view-if-needed/dist/index.js 37:26-27
export 'compute' (imported as 't') was not found in 'compute-scroll-into-view' (possible exports: default)

ERROR in ./node_modules/scroll-into-view-if-needed/dist/index.js 39:45-46
export 'compute' (imported as 't') was not found in 'compute-scroll-into-view' (possible exports: default)

webpack 5.88.2 compiled with 2 errors and 392 warnings in 87052 ms
```

How can I fix it?
There is my [webpack.config.js](https://gist.github.com/daydarkln/ab2046c11ff2eff63f933d5b62d38b71#file-webpack-config-js)

I tried to use this issue, but it doesn't help: https://github.com/scroll-into-view/scroll-into-view-if-needed/issues/1181

I tried to change react/wepback/[babel](https://gist.github.com/daydarkln/ab2046c11ff2eff63f933d5b62d38b71#file-babel-config-js) versions, but it doesn't help too

What didn't I notice?

**Comment by afc163** [comment]
https://github.com/scroll-into-view/scroll-into-view-if-needed/issues/1163#issuecomment-1501054052
